CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKSwapChain.java

```diff
@@ -654,14 +726,11 @@ private void DestroyVKSwapChain() {
                 
         // The swapchain creates its own images but we create the image views for each one, destroy those now
         for (int i = 0; i < imageCount; ++i) {
-            vkDestroyImageView(cvkDevice.GetDevice(), imageViewHandles.get(i), null);
-            imageViewHandles.set(i, VK_NULL_HANDLE);
+            swapChainImages.get(i).DestroyImageView();
```

Comment:
Maybe DestroyImageView and ResetImageHandle should be called from the Destroy of CVKSwapChain image?  If it has it's own destroy it can make sure to not destroy the image and image memory.

Reply:
Changed